depext.ml

```diff
@@ -41,11 +41,11 @@ let string_split char str =
   aux 0
 
 let has_command c =
-  let cmd = Printf.sprintf "/bin/sh -c command -v %s" c in
+  let cmd = Printf.sprintf "command -v %s" c in
```

Comment:
`command` is POSIX, but I can believe that it's not portable despite that.  Would it be possible for `cmdliner` to expose the `find_cmd` logic so that we dont have to cut and paste it here?